### PR TITLE
Updating of readme, suggesting fixes

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +69,4 @@ wafConfig.disabledRuleGroups.push({
 networkManagementClient.applicationGateways.createOrUpdate(resourceGroup, appGatewayName, result, _);
 ```
 
-Once again, some parts of this code you can see in auto generated commands.
```
Comment (number213) [inline README.md]: I don't like this line to, but I'd rather keep it and elaborate that code that retrieves and updates WAF config is present in auto generated files, so you can use it as a reference or copy-paste and modify.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,18 @@
 # Limitations of code generation & how to overcome them
 
-Unfortunately, generator can not generate code for processing complex properties such as an array of objects. In that case developer has to extend auto generated code by adding handwritten code for processing complex properties. Following text will explain how to write hybrid solution using Web Application Firewall Config as an example.
+Unfortunately, generator can not generate code for processing of complex properties such as an array of objects. In that case developer has to extend auto generated code by adding handwritten code for processing of complex properties. Let's call this hybrid solution i.e. partially generated and partially handwritten. This manual explains how to write hybrid solution using Web Application Firewall Config as an example.
```
Comment (number213) [inline README.md]: Agreed, it should be explained what term `hybrid solution` means.

P.S. I'm pretty sure you don't need to add "of" to `gerund as a verb + noun` construction.

Comment (albonemo) [inline README.md]: "Unfortunately"  and reapiting "processing of complex properties" lets cut out, mske something like 
"Generator can not generate code for processing complex properties such as an array of objects and developer has to extend auto generated code by adding handwritten code for them."

File: README.md [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
 
-*Please make sure you've read [documentation](https://github.com/Azure/azure-xplat-cli/blob/dev/Documentation/) (especially [the part about writing cmdlets](https://github.com/Azure/azure-xplat-cli/blob/dev/Documentation/Writing-Cmd.md)) for `azure-xplat-cli`*
+*Please compete reading of [documentation](https://github.com/Azure/azure-xplat-cli/blob/dev/Documentation/) (especially [the part about writing cmdlets](https://github.com/Azure/azure-xplat-cli/blob/dev/Documentation/Writing-Cmd.md)) for `azure-xplat-cli` before continuing*
```
Comment (albonemo) [inline README.md]: "before continuing" - I think that's bad idea to interupt readin by sennding to another doc.
Better "write please, read doc to know how to"

**Pull request merged**